CHANGELOG.md

```diff
@@ -1,26 +1,48 @@
+- This release requires a minimum Go version of v1.22.7 or higher.
+- The `releases` branch has been deprecated with the 202408 split of 'Lotus Node' and 'Lotus Miner'. See https://github.com/filecoin-project/lotus/blob/master/LOTUS_RELEASE_FLOW.md#why-is-the-releases-branch-deprecated-and-what-are-alternatives for more info and alternatives for getting the latest release for both the 'Lotus Node' and 'Lotus Miner' based on the Branch and Tag Strategy.
+   - To get the latest Lotus Node tag: git tag -l 'v*' | sort -V -r | head -n 1
+   - To get the latest Lotus Miner tag: git tag -l 'miner/v*' | sort -V -r | head -n 1
 
-## Bug Fixes
+## The Filecoin network version 24 delivers the following FIPs:
+
+- [FIP-0081: Introduce lower bound for sector initial pledge](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0081.md)
+- [FIP-0086: Fast Finality in Filecoin (F3)](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0086.md)
+- [FIP-0094: Add Support for EIP-5656 (MCOPY Opcode) in the FEVM](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0094.md)
+- [FIP-0095: Add FEVM precompile to fetch beacon digest from chain history](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0095.md)
+
+## v15 Builtin Actor Bundle
+
+This release candidate uses the [v15.0.0-rc1](https://github.com/filecoin-project/builtin-actors/releases/tag/v15.0.0-rc1)
+
+## Migration
+
+All node operators, including storage providers, should be aware that ONE pre-migration is being scheduled 120 epochs before the network upgrade. The migration for the NV24 upgrade is expected to be light with no heavy pre-migrations, here are some expected timings and resource consumption numbers:
+
+- Pre-Migration is expected to take less then 1 minute.
+- The migration on the upgrade epoch is expected to take less than 30 seconds on a node with a NVMe-drive and a newer CPU. For nodes running on slower disks/CPU, it is still expected to take less then 1 minute.
+- RAM usages is expected to be under 20GiB RAM for both the pre-migration and migration.
+
+We recommend node operators (who haven't enabled splitstore discard mode) that do not care about historical chain states, to prune the chain blockstore by syncing from a snapshot 1-2 days before the upgrade.
+
+For certain node operators, such as full archival nodes or systems that need to keep large amounts of state (RPC providers), we recommend skipping the pre-migration and run the non-cached migration (i.e., just running the migration at the network upgrade epoch), and schedule for some additional downtime. Operators of such nodes can read the [How to disable premigration in network upgrade tutorial.](https://lotus.filecoin.io/kb/disable-premigration/)
+
+## 📝 Changelog
 
-## Deps
+See https://github.com/filecoin-project/lotus/compare/v1.29.2...release/v1.30.0-rc1 for the set of changes since the last stable release.
```

Comment:
```suggestion
See https://github.com/filecoin-project/lotus/compare/v1.29.2...release/v1.30.0 for the set of changes since the last stable release.
```

If comparing to a branch, then it needs to be `release/v1.30.0`.  If comparing to a tag, then `v1.30.0-rc1`.

This is an area where assuming we break out node changelog entry vs. a miner release changelog we'll need to compare to the right version (relates to the discussion in https://github.com/filecoin-project/lotus/pull/12562/files#r1790984626 ).  I assume we would do: https://github.com/filecoin-project/lotus/compare/miner/v1.28.2...release/miner/v1.29.0

Reply:
```suggestion
For the set of changes since the last stable release:
* Node: https://github.com/filecoin-project/lotus/compare/v1.29.2...v1.30.0-rc1
* Miner: https://github.com/filecoin-project/lotus/compare/v1.28.3...miner/v1.30.0-rc1
```

I assume we compare to the tag so that we're fixed in time.